dot/rpc/modules/system_integration_test.go

```diff
@@ -61,11 +62,12 @@ var (
 func newNetworkService(t *testing.T) *network.Service {
 	testDir := path.Join(os.TempDir(), "test_data")
 
+	ctrl := gomock.NewController(t)
 	cfg := &network.Config{
-		BlockState:         network.NewMockBlockState(nil),
+		BlockState:         network.NewMockBlockState(ctrl),
 		BasePath:           testDir,
-		Syncer:             network.NewMockSyncer(),
-		TransactionHandler: network.NewMockTransactionHandler(),
+		Syncer:             network.NewMockSyncer(ctrl),
+		TransactionHandler: network.NewMockTransactionHandler(ctrl),
```

Comment:
Do you need to initialize all this actually if they don't expect anything? Can't these be left to `nil`?